src/shared_modules/utils/timeHelper.h

```diff
@@ -19,6 +19,22 @@
 #include <sstream>
 #include <string>
 
+#ifdef WIN32
+
+static struct tm* gmtime_r(const time_t* timep, struct tm* result)
+{
+    errno = gmtime_s(result, timep);
+    return errno == 0 ? result : nullptr;
+}
+
+static struct tm* localtime_r(const time_t* timep, struct tm* result)
+{
+    errno = gmtime_s(result, timep);
```

Comment:
shouldn't this be localtime_s instead of gmtime_s?

Reply:
It's a typo. Fixed.